sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/models/DigitalTwinsJsonPropertyNames.java

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.digitaltwins.core.models;
+
+/**
+ * String constants for use in JSON de/serialization for digital twins, digital twin relationships,
+ * and digital twin components.
+ */
+public class DigitalTwinsJsonPropertyNames {
+    /**
+     * The JSON property name for the Id field on a digital twin.
+     */
+    public static final String DIGITAL_TWIN_ID = "$dtId";
+
+    /**
+     * The JSON property name for the ETag field on a digital twin.
+     */
+    public static final String DIGITAL_TWIN_E_TAG = "$etag";
+
+    /**
+     * The JSON property name for the metadata field on a digital twin.%
```

Comment:
Also useful for a component

Reply:
I got these from your code, FYI. You should update the .NET comments to match

Reply:
Will do, thanks.